viewer/packages/rendering/src/glsl/pointcloud/normalize.frag

```diff
@@ -3,23 +3,48 @@ precision highp float;
 uniform sampler2D tDepth;
 uniform sampler2D tDiffuse;
 
+#if defined(use_edl)
+	#pragma glslify: import('./edl.glsl');
+
+	uniform float screenWidth;
+	uniform float screenHeight;
+	uniform vec2 neighbours[NEIGHBOUR_COUNT];
+	uniform float edlStrength;
+	uniform float radius;
+
+#endif
+
 in vec2 vUv;
 
 out vec4 outputColor;
 
 void main() {
-    float depth = texture(tDepth, vUv).r;
-
-	if(depth >= 1.0){
-		discard;
-	}
-
+    
+	float depth = texture(tDepth, vUv).r;
     vec4 color = texture(tDiffuse, vUv);
+	bool shouldDiscard = depth >= 1.0;
```

Comment:
Can this ever be true?

Reply:
Yes, if you remove this condition, background is always rendered white. I am not sure about why but I think it's because of the values that are written to the point cloud render target.

Reply:
Can confirm what Savelii said. Also, on my computer it works to make the comparison `depth == 1.0`, but probably a good idea to keep it as-is in case of flaky implementations.